buildpack/telemetry/metrics.py

```diff
@@ -108,9 +108,18 @@ def _micrometer_runtime_requirement(runtime_version):
 
 def micrometer_metrics_enabled(runtime_version):
     """Check for metrics from micrometer."""
-    return bool(get_micrometer_metrics_url()) and _micrometer_runtime_requirement(
-        runtime_version
-    )
+    logging.info("checking is micrometer metrics enabled")
+    micrometer_enabled=False
+    if(_micrometer_runtime_requirement(runtime_version)):
+        logging.info("satisfies micrometer runtime requirement")
+        if(bool(get_micrometer_metrics_url())):
+            logging.info("Found micrometer metrics url configured")
+            micrometer_enabled = True
+        elif(strtobool(os.getenv("NON_MENDIX_PUBLIC_CLOUD","false"))):
+            logging.info("micrometer for non mendix public cloud")
+            micrometer_enabled =  True
+    return micrometer_enabled
+    
```

Comment:
Not sure if we need these many `INFO` log lines here. Can we have it as `DEBUG` instead?

Reply:
Changed to debug. Thanks!